daemon/tests/harness/mod.rs

```diff
@@ -292,6 +293,7 @@ impl Taker {
             Duration::from_secs(10),
             projection_actor,
             maker_identity,
+            PathBuf::from_str("/dev/null").expect("Valid path string"),
```

Comment:
Not sure what is the best way to deal with this.

Reply:
Yeah that is fine, wallet is mocked. You can use `from` though to avoid the error:

```suggestion
            PathBuf::from("/dev/null"), // wallet is mocked in tests, path doesn't matter
```